app/models/work_version.rb

```diff
@@ -123,4 +123,11 @@ def published_edtf
   def created_edtf
     EDTF.parse(super)
   end
+
+  sig { returns(T.nilable(String)) }
+  def description
+    return nil unless %w[version_draft].include? state
```

Comment:
Can you say more about what this method is doing?

Reply:
Yes, I can add a comment if that would help. Essentially, we do not want the description field to be populated when editing a work that isn't in a draft state. If we're opening an object in a draft state, whatever was entered in the what's changing box as part of the earlier edit, we do want to populate that. But if one is starting a new version, we want the field blank.

Reply:
OK, gotcha. What would you think about nulling it out when new versions are instantiated? Like here: https://github.com/sul-dlss/happy-heron/blob/3e679fd9a5299035c1a90866f52897665af85b95/app/controllers/works_controller.rb#L139-L145 with something like

```ruby
work_version.description = nil
```

Reply:
Let me know if I'm grokking. I may be turned around about this. Happy to pair this afternoon too if you have time. I'm free all PM.

Reply:
I don't think that will work as, from my understanding, the new version is not initiated the form is edited and save or save as draft is clicked... at least that's what I was able to see in my local testing. Is that incorrect?

Reply:
I'm not sure I'm following. :pear: ?